README.md

```diff
@@ -245,21 +245,21 @@ full_story_org_id: "org1123"
 
 ## Customizing the code
 
-In addition to customization via configuration, you can customize Vectara Answer further by modifying it's code directly in your own fork of the repository.
+In addition to customization via configuration, you can customize Vectara Answer further by modifying its code directly in your own fork of the repository.
 
 ### Modifying the UI
 
 The UI source code is all in the `src/` directory. See the [UI README.md](https://github.com/vectara/vectara-answer/blob/main/src) to learn how to make changes to the UI source.
 
 ### Modifying the Proxy Server
 
-While the app run via `npm run start` works with a local client that accesses the Vectara API directly, running the app via Docker (see below) spins up a full-stack solution, using a proxy server to make Vectara API requests.
```

Comment:
Perhaps rewrite as "While `npm run start` runs the application with a local client..."

Reply:
Yeah, it can be done 👍. No problem 😁.